src/java/com/puppetlabs/ssl_utils/SSLUtils.java

```diff
@@ -189,28 +196,43 @@ public static X509Certificate signCertificate(String issuerDn,
                                                   String subjectDn,
                                                   PublicKey subjectPublicKey,
                                                   List<Map<String, Object>> extensions)
-            throws IOException, OperatorCreationException, CertificateException, NoSuchAlgorithmException, SignatureException, InvalidKeyException {
-        X509V3CertificateGenerator certGen = new X509V3CertificateGenerator();
```

Comment:
Oh sweet, so we were able to get rid of using this deprecated class? Finally. Looks like we can get rid of the import statement for this class (surprised IDEA didn't take care of that).